templates/include/glean/glean.1.schema.json

```diff
@@ -36,8 +36,6 @@
         "app_build",
         "app_display_version",
         "architecture",
-        "device_manufacturer",
```

Comment:
@badboy can you kindly also update the docs [here](https://mozilla.github.io/glean/book/user/pings/index.html#the-client_info-section) and the related `metrics.yaml` files once that's merged?

Reply:
of course!